src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -25,7 +25,7 @@
 
 /*
  * =======================================================================
- * (c) Copyright IBM Corp. 2018, 2022 All Rights Reserved
+ * (c) Copyright IBM Corp. 2018, 2024 All Rights Reserved
  * =======================================================================
```

Comment:
Please update the copyright format - four more `=` on lines 27 and 29, like was added to `ModuleWriter.java`.

Reply:
Updated